src/app/app-routing.module.ts

```diff
@@ -1,7 +1,12 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import {RegisterComponent} from "./auth/register/register.component";
+import {LoginComponent} from "./auth/login/login.component";
 
-const routes: Routes = [];
+const routes: Routes = [
```

Comment:
For now this totally fine, but after you add a Home page, please replace it to the default route